safe-global/web-core
# [Create Safe] feat: Connect wallet step

## What it solves

Resolves #1003 

## How this PR fixes it

- Adds a first step to the create safe flow for connecting a wallet
- Disables the Continue button if the wallet is disconnected

## How to test it

1. Open /create-safe while disconnected
2. A button to connect the wallet exists
3. A QR code for mobile pairing exists
4. After connection it should automatically move to the next step

## Screenshots
<img width="840" alt="Screenshot 2022-10-31 at 08 44 00" src="https://user-images.githubusercontent.com/5880855/198956985-f0ab612c-cc16-4777-9025-10d7bc953b9c.png">
